lab-trent/README.md

```diff
@@ -0,0 +1,2 @@
+# About
```

Comment:
Moar! 

Specifically, go into more detail about how to use the app. How to I install it, run it, make calls to a route, which route(s), etc.